Enterprise-CMCS/cmcs-eregulations
# Eregcsc 2016 deploy eua changes prod

Resolves #EREGCSC-2016

**Description-**
Now that we are going to use EUA in production we need to update our base.py settings to contain all the EUA environment variables. 

**This pull request changes...**
base.py will be calling euasettings.py
euasettings.py will be called by base.py. 
- expected change 2

**Steps to manually verify this change...**

Steps to view and verify change
1. Visit the admin page and sign in with EUA
2. Ensure singing in with EUA still works.
